source/adminguide/networking/advanced_zone_config.rst

```diff
@@ -123,13 +128,17 @@ Configuring a Shared Guest Network
          created for. You must specify the domain the project belongs
          to.
 
-      -  **All**: The guest network is available for all the domains,
-         account, projects within the selected zone.
+      -  **All**: (Administrators only) The guest network is available
+         for all the domains, account, projects within the selected zone.
 
    -  **Network Offering**: If the administrator has configured multiple
       network offerings, select the one you want to use for this
       network.
 
+   -  **Associated Network**: The L2 or Isolated network this network is
```

Comment:
Should this be here as well or only on Add new VPC gateway?

Reply:
@nvazquez 
associated network is available for both user-shared network and user-vpc private gateway.